pkg/cliplugins/workspace/plugin/kubeconfig.go

```diff
@@ -298,9 +305,51 @@ func (o *UseWorkspaceOptions) Run(ctx context.Context) error {
 		return err
 	}
 
+	if err := o.getAPIBindings(ctx, newServerHost); err != nil {
+		// display the error, but don't stop the current workspace from being reported.
+		_, _ = fmt.Fprintf(o.Out, "error checking APIBindings: %v", err)
+	}
+
 	return currentWorkspace(o.Out, newServerHost, shortWorkspaceOutput(o.ShortWorkspaceOutput), workspaceType)
 }
 
+// getAPIBindings retrieves APIBindings within the workspace and look for ones without exported PermissionClaims.
+func (o UseWorkspaceOptions) getAPIBindings(ctx context.Context, host string) error {
+	_, clusterName, err := pluginhelpers.ParseClusterURL(host)
+	if err != nil {
+		return err
+	}
+
+	apiBindings, err := o.kcpClusterClient.Cluster(clusterName).ApisV1alpha1().APIBindings().List(ctx, metav1.ListOptions{})
+	if err != nil && !apierrors.IsNotFound(err) {
+		return err
+	} else if apierrors.IsNotFound(err) {
+		// nothing to do if there aren't any
+		return nil
+	}
+
+	for _, binding := range apiBindings.Items {
+		for _, exportedClaim := range binding.Status.ExportPermissionClaims {
+			found := false
+			for _, specClaim := range binding.Spec.PermissionClaims {
+				if exportedClaim.Equal(specClaim.PermissionClaim) {
+					found = true
+					break
+				}
+			}
+			if !found {
+				group := exportedClaim.Group
+				if group == "" {
+					// Print the quotes for built-in types to make the output more decipherable.
+					group = `""`
+				}
+				fmt.Fprintf(o.Out, "Warning: claim for [Group:%s, Resource: %s] exported but not specified on APIBinding %s\nAdd this claim to the APIBinding's Spec.\n", group, exportedClaim.Resource, binding.Name)
```

Comment:
Kubernetes users are familiar with group and resource being formatted as `resource.group` e.g. `rolebinding.authorization.rbac.k8s.io` -

Reply:
This is true, however formatting it this way with a core type such as configmaps results in `configmaps.`. Do you think this is an edge case here?

Reply:
Ok - I'll change this and make sure we're formatting core resources correctly.

Reply:
If you create a `schema.GroupResource` object and use the built-in formatters IIRC this is all handled for ya

Reply:
@stevekuznetsov It is, but I was using https://github.com/kcp-dev/kcp/blob/main/pkg/apis/apis/v1alpha1/types_apiexport.go#L199 :)